api/lib/domain/usecases/update-organization-learner-dependent-user-password.js

```diff
@@ -36,9 +36,10 @@ const updateOrganizationLearnerDependentUserPassword = async function ({
   const generatedPassword = passwordGenerator.generateSimplePassword();
   const hashedPassword = await cryptoService.hashPassword(generatedPassword);
 
-  await authenticationMethodRepository.updatePasswordThatShouldBeChanged({
+  await authenticationMethodRepository.updatePassword({
     userId: userStudent.id,
     hashedPassword,
+    shouldChangePassword: true,
```

Comment:
Pourrait-on commencer à renommer les `shouldChangePassword` en `mustChangePassword` ? En effet il s'agit d'une obligation absolument non-optionnelle à laquelle l'utilisateur ne peut échapper.

```suggestion
    mustChangePassword: true,
```

Reply:
J'ai tenté de le faire en partie, mais le fait d'avoir `mustChangePassword` et parfois `shouldChangePassword` est troublant et entraine de la confusion. Car on écrit dans les repo en utilisant `mustChangePassword`, on stocke `shouldChangePassword` en base et on consomme `shouldChangePassword`.

Je pense qu'il faudrait faire ce travail de manière plus global dans une PR dédiée, sinon on risque d'avoir cet état partiel (et confus) pendant un moment.

Reply:
Bien reçu. Ça me convient bien. Ma remarque et ton essai ont eu le mérite de remettre ce point sur la table. Merci.